patches/lighthouse+12.2.2.patch

```diff
@@ -1,11 +1,12 @@
 diff --git a/node_modules/lighthouse/core/index.d.cts b/node_modules/lighthouse/core/index.d.cts
-index 1c399e1..23c3d1f 100644
+index a3645a1..b105ca3 100644
 --- a/node_modules/lighthouse/core/index.d.cts
 +++ b/node_modules/lighthouse/core/index.d.cts
-@@ -1,4 +1,6 @@
- export = lighthouse;
- /** @type {import('./index.js')['default']} */
+@@ -7,5 +7,7 @@ export = lighthouse;
+  * @property {import('./index.js')['snapshot']} snapshot
+  */
+ /** @type {import('./index.js')['default'] & ExportType} */
 +// Otherwise TS is confused when using ES types in CJS.
 +// @ts-ignore
- declare const lighthouse: typeof import("./index.js")['default'];
+ declare const lighthouse: typeof import("./index.js")["default"] & ExportType;
```

Comment:
This patch was still necessary? That's too bad… it would be nice to dig into why it's needed and try to address that.